compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Monoid.hs

```diff
@@ -11,11 +11,59 @@ module DA.Daml.Doc.Render.Monoid
 import DA.Daml.Doc.Types
 import Control.Monad
 import Data.Foldable
+import Data.Maybe
+import Data.List.Extra
 import System.FilePath
 import qualified Data.Map.Strict as Map
 import qualified Data.Set as Set
 import qualified Data.Text as T
 
+data RenderOut
+    = RenderSpaced [RenderOut]
+    | RenderModuleHeader T.Text
+    | RenderSectionHeader T.Text
+    | RenderAnchor Anchor
+    | RenderBlock RenderOut
+    | RenderList [RenderOut]
+    | RenderFields [(RenderText, RenderText, RenderText)]
+    | RenderPara RenderText
```

Comment:
Seem slightly weird to abbreviate this, if none of the other record fields are abbreviated and this is not the longest. But I didn’t get confused by the abbreviation so I’m fine with keeping it.